tests/models/coreference_resolution/coref_test.py

```diff
@@ -23,21 +23,30 @@ def test_decode(self):
                                   [5, 6],
                                   [14, 56],
                                   [17, 80]])
-        spans = Variable(spans.unsqueeze(0))
 
-        # Indices into ``spans`` indicating that the
-        # two mentions are co-referent.
-        antecedents = torch.LongTensor([-1, 0, -1, -1, 1, 3])
-        antecedents = Variable(antecedents.unsqueeze(0))
+        antecedent_indices = torch.LongTensor([[0, 0, 0, 0, 0, 0],
```

Comment:
Are these 1-indexed?

Reply:
I think so - it's an edge case because the model will always learn to predict that the first mention can't have an antecedent (i.e it always predicts -1), so I don't exactly know what the behaviour is here. @kentonl?

Reply:
Yes, they are zero-indexed. Even though the scores are prepended with zeros due to the dummy antecedent, [this line](https://github.com/DeNeutoy/allennlp/blob/c2289bf210186dcaf6ffe68e447a77446b70baa3/allennlp/models/coreference_resolution/coref.py#L558) shifts all the indices back, and the dummy antecedent has an index of -1.